package.json

```diff
@@ -8,8 +8,8 @@
     "format": "prettier \"src/main/webapp/**/*.(js|jsx|css)\" --write",
     "format:verify": "prettier \"src/main/webapp/**/*.(js|jsx|css)\" --list-different",
     "eslint": "eslint src --ext \".js,.jsx\"",
-    "eslint:ci": "yarn run eslint -f junit -o test-reports/lint-results.xml",
-    "eslint:fix": "eslint src --ext \".js, .jsx\" --fix",
+    "eslint:fix": "yarn run eslint --fix",
+    "lint": "yarn run eslint && yarn run format:verify",
```

Comment:
last check as i'm not that familiar with that repo yet, maybe it would be worth to leave that eslint:ci as a pipeline just in case it is necessary somewhere

Reply:
nah, we will know faster if we break it. It was used in github action, but I edited it